src/QirTools/pyqir/pyqir/__init__.py

```diff
@@ -1,59 +1,214 @@
+# Copyright(c) Microsoft Corporation.
+# Licensed under the MIT License.
+
 from .pyqir import *
 
+
 class QirBuilder:
+    """
+    The QirBuilder object describes quantum circuits and emits QIR
```

Comment:
```suggestion
    The QirBuilder object describes quantum circuits and emits QIR.
```

It may also be helpful to reference the QIR Base Profile explicitly here instead of implicitly via the word "circuit." (Typically, a "quantum circuit" is a nonadaptive quantum program, but I think we mean something more specific in this context?)